easybuild/easyconfigs/p/Python/Python-2.7.14-foss-2017b.eb

```diff
@@ -0,0 +1,234 @@
+name = 'Python'
+version = '2.7.14'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['a4f05a0720ce0fd92626f0278b6b433eee9a6173ddf2bced7957dfb599a5ece1']
```

Comment:
WIP since final release of Python 2.7.14 is not there yet, it's planned for Sept 13th (cfr. https://mail.python.org/pipermail/python-dev/2017-August/148864.html).

Obviously the checksums will need to be updated...